androidApp/src/main/java/press/IntentReceiverActivity.kt

```diff
@@ -0,0 +1,20 @@
+package press
+
+import android.app.Activity
+import android.os.Bundle
+import press.editor.EditorActivity
+
+class IntentReceiverActivity : Activity() {
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    processIntent()
+
+  }
+
+  private fun processIntent() {
+
+    val intent = EditorActivity.intent(this)
+    startActivity(intent)
+    finish()
+  }
+}
```

Comment:
The spacing in this file is a bit off. I am adding `ktlint` so that PR authors don't have to deal with formatting comments anymore. This file should get flagged once the CI checks run again.